doc/conf.py

```diff
@@ -65,7 +72,23 @@
       if os.path.exists(inv_file):
         haddock_mapping[entry.name] = (html_dir, inv_file)
 
-haddock_mapping['plutus'] = ('https://plutus.readthedocs.io/en/latest', os.getenv('PLUTUS_CORE_OBJECTS_INV', None))
+# COMBINED_PLUTUS_CORE_HADDOCK comes from the `combined-plutus-core-haddock.nix` derivation.
+# We populate haddock_mapping taking care not to override existing packages.
+# Note that this means that plutus-core haddock is embedded into plutus-apps's read-the-docs site.
+haddock_dir = os.getenv('COMBINED_PLUTUS_CORE_HADDOCK', None)
```

Comment:
why not merge this  block with the one above and just looping over a list of two haddock directories? It's the same code